recipes/mrmr_selection/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "mrmr_selection" %}
+{% set version = "0.2.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # no sdist provided, no tags on github
+  # see issue: https://github.com/smazzanti/mrmr/issues/45
```

Comment:
If you believe that the wheel was created after this commit,

https://github.com/smazzanti/mrmr/commit/3ea01aac88628b678aa88f734524594b0edd63d7

we could use that instead. At leas it would give us a traceable source for it.

PS: Philosophical questions.  If a package doesn't publish an sdist or tag to a released wheel, is it really open?